docs/technical-reference.md

```diff
@@ -287,24 +287,40 @@ and hybrid flows, and check the box to issue ID tokens
 
 ### Okta
 
-1. Go to Okta > Admin > Applications
-1. Click `Add Application`
-1. Select type `Web` 
-1. Name the app `Quilt` or something similar
-1. Configure the app as shown below
-1. Add `<QuiltWebHost>` to `Login redirect URIs` and
-`Initiate login URI`
-1. Copy the `Client ID` to a safe place
-1. Go to API > Authorization servers
-1. You should see a default URI that looks something like the following:
+Note: You will need Okta administrator privileges to add a new Application.
+
+1. Go to **Okta > Admin > Applications > Applications**
```

Comment:
For style consistency with the rest of docs, no bold here.

Reply:
Bold removed